### PR TITLE
Update efa-cheatsheet.md

### PR DESCRIPTION
**Pull request by awsankur**
Added note on NVLinks for P5s

*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: 1.architectures/efa-cheatsheet.md [hunk 1]
```diff
@@ -26,6 +26,9 @@ versions of your libfabric.
 Use cuda>=12.0, nccl>=2.18.0 (recommend at least 2.18.5), aws-ofi-nccl>=1.7.2 (recommend at least
 1.7.3).
 
+The dcgm command to validate the NVLinks is `sudo dcgmi diag -r 2 -p pcie.gpu_nvlinks_expected_up=18` where the 18 is H100 specific. 
+`nvidia-smi nvlink -s`  is the command to get the status for all NVLinks for each of the GPUs. for H100 there are 18
```
Comment (sean-smith) [inline 1.architectures/efa-cheatsheet.md:30]: can you include a table with number of links per gpu? i.e.

| Instance | GPU  | Nvlink |
|----------|------|--------|
| p4d(e)   | A100 | 4      |
| p5       | H100 | 18     |

Also can you put the command with sample output?

Comment (mhuguesaws) [reply 1.architectures/efa-cheatsheet.md:30]: and add the NVSwitch generation https://www.nvidia.com/en-us/data-center/nvlink/

Comment (perifaws) [reply 1.architectures/efa-cheatsheet.md:30]: @awsankur proceed to the recommended changes?

**Review by mhuguesaws — APPROVED** [review]
LGTM

**Pull request merged**